packages/owl-bot/test/fake-configs-store.ts

```diff
@@ -0,0 +1,86 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import {Configs, ConfigsStore} from '../src/configs-store';
+import {OwlBotLock, toFullMatchRegExp} from '../src/config-files';
+import {GithubRepo, githubRepoFromOwnerSlashName} from '../src/github-repo';
+// There are lots of unused args on fake functions, and that's ok.
+/* eslint-disable @typescript-eslint/no-unused-vars */
+
+export class FakeConfigsStore implements ConfigsStore {
+  readonly configs: Map<string, Configs>;
+  readonly githubRepos: Map<string, GithubRepo>;
+
+  constructor(configs?: Map<string, Configs>) {
+    this.configs = configs ?? new Map<string, Configs>();
+    this.githubRepos = new Map();
+  }
+  findReposAffectedByFileChanges(
```

Comment:
I like that you've broken out a `FakeConfigsStore` class for testing. I might be tempted to pull this into a `mocks/` or `utils/` subfolder, since it's not itself testing anything.